lib/ansible/modules/package_facts.py

```diff
@@ -440,7 +440,7 @@ class APK(CLIMgr):
 
     def list_installed(self):
         rc, out, err = module.run_command([self._cli, 'info', '-v'])
-        if rc != 0 or err:
+        if rc != 0:
```

Comment:
This change isn't in devel. Why wasn't this change made there first?

Also, what's the relationship between this change and the integration test changes?

Reply:
The change is available in `devel` via https://github.com/ansible/ansible/pull/83132 

The integration test makes sure that package fact dict is populated `ansible_facts.packages | length > 1`

Reply:
Ah, I had missed that since the module change and the tests were from separate PRs. Thanks.

Reply:
No problem. Thanks.